### PR TITLE
Mosaic primitive

### PR DESCRIPTION
**Pull request by TolyaTalamanov**

### CODE REVIEW

**Comment by TolyaTalamanov** [comment]
@smirnov-alexey @dmatveev

File: modules/gapi/src/api/render_ocv.cpp [hunk 1]
```diff
@@ -79,10 +79,16 @@ void poly(cv::Mat mat, std::vector<cv::Point>, cv::Scalar color, int lt, int shi
 
 void mosaic(cv::Mat mat, const cv::Rect &rect, int cellSz)
 {
-    auto mos = mat(rect);
-    cv::Mat tmp;
-    cv::resize(mos, tmp, {mos.size().width/cellSz, mos.size().height/cellSz}, 0, 0, cv::INTER_LINEAR);
-    cv::resize(tmp, mos, {rect.width, rect.height}, 0, 0, cv::INTER_LINEAR);
+    cv::Mat msc_roi = mat(rect);
+    int crop_x = msc_roi.cols - msc_roi.cols % cellSz;
```
Comment (smirnov-alexey) [inline modules/gapi/src/api/render_ocv.cpp:83]: Maybe it's better to not handle cases where `msc_roi.cols % cellSz` != 0?

Comment (TolyaTalamanov) [reply modules/gapi/src/api/render_ocv.cpp]: OSD handle it

Comment (smirnov-alexey) [reply modules/gapi/src/api/render_ocv.cpp:83]: Nevermind

**Review by smirnov-alexey — APPROVED** [review]

**Pull request merged**